chapters/zh/chapter1.md

```diff
@@ -41,6 +41,13 @@ id: 1
 
 <codeblock id="01_02_03"></codeblock>
 
+### 第四部分: 中文
```

Comment:
We can probably remove one of the other examples here and start with Chinese 😃 So it could be: Chinese, English and either German or French, or any other language spaCy supports (whatever you think is more interesting for Chinese users).

Reply:
Chinese, English and German/French sounds cool.
Do I understand correctly that I only have to change the codeblock id in chapter1.md here, and no need to change the code files themselves?